### PR TITLE
Update hyperkit go driver.

### PR DESCRIPTION
**Pull request by dlorenc**
Some patches didn't apply cleanly, not sure what the problem was.

### CODE REVIEW

File: vendor/k8s.io/apimachinery/pkg/apimachinery/registered/registered.go [hunk 1]
```diff
@@ -242,7 +242,12 @@ func (m *APIRegistrationManager) RESTMapper(versionPatterns ...schema.GroupVersi
 	for enabledVersion := range m.enabledVersions {
 		if !unionedGroups.Has(enabledVersion.Group) {
 			unionedGroups.Insert(enabledVersion.Group)
-			groupMeta := m.groupMetaMap[enabledVersion.Group]
+			groupMeta, found := m.groupMetaMap[enabledVersion.Group]
```
Comment (r2d4) [inline vendor/k8s.io/apimachinery/pkg/apimachinery/registered/registered.go]: It's possible that this patch wasn't applied properly during the last merge.  It shouldn't really matter now that TPRs are deprecated.

**Comment by dlorenc** [comment]
Re-added context.

**Comment by dlorenc** [comment]
@minikube-bot test this please

**Comment by dlorenc** [comment]
@minikube-bot test this please

**Pull request merged**